src/lsm/lsm_cursor.c

```diff
@@ -392,6 +393,7 @@ __clsm_open_cursors(
 	txn = &session->txn;
 	lsm_tree = clsm->lsm_tree;
 	chunk = NULL;
+	close_range_end = close_range_start = 0;
```

Comment:
I'd feel more comfortable if we move this down near the other cases.  I don't know for sure, but the retry loop in this function scares me: it looks like we could repeatedly try to close cursors?